internal/mode/static/state/dataplane/configuration.go

```diff
@@ -455,6 +455,11 @@ func createHTTPFilters(filters []v1.HTTPRouteFilter) HTTPFilters {
 				// using the first filter
 				result.RequestRedirect = convertHTTPRequestRedirectFilter(f.RequestRedirect)
 			}
+		case v1.HTTPRouteFilterURLRewrite:
```

Comment:
I think we are missing some validation here. From the spec:

> All filters are expected to be compatible with each other except for the URLRewrite and RequestRedirect filters, which may not be combined. If an implementation can not support other combinations of filters, they must clearly document that limitation. In cases where incompatible or unsupported filters are specified and cause the Accepted condition to be set to status False, implementations may use the IncompatibleFilters reason to specify this configuration error.

and:

> HTTPURLRewriteFilter defines a filter that modifies a request during forwarding. At most one of these filters may be used on a Route rule. This MUST NOT be used on the same Route rule as a HTTPRequestRedirect filter.

It seems like we should be setting `Accepted/False/IncompatibleFilters` if both `HTTPRequestRedirect` and `HTTPURLRewriteFilter` are set.

Reply:
CEL denies this from even happening in the first place.

Reply:
Hmm ok. I guess we would only need to set that condition if there were other incompatible filters.